client/my-sites/hosting/controller.js

```diff
@@ -30,6 +31,22 @@ function waitForState( context ) {
 	} );
 }
 
+export async function redirectIfInsufficientPrivileges( context, next ) {
```

Comment:
What do you think about following the current naming convention patterns e.g. `redirectIfCant*` or just `redirectIfNoAccess`?

Reply:
I will use the refactor in https://github.com/Automattic/wp-calypso/pull/85914/commits/ebf7e623a57ec556b668f4886e171189bf88cdfb once it's ready.